### ISSUE TITLE
Nemo automatically generates (unwanted) bookmarks

### ISSUE THREAD

**Issue by Grydpype** [linuxmint/nemo#3475]
### Distribution

Mint 22

### Package version

6.2.8

### Frequency

Always

### Bug description

I have some of my data in a Cryptomator vault called **Erasmus** which is opened automatically after a login. 
In the course of a session Nemo creates many identical bookmark pointing to the open vault's root directory in _/home/.local/share/Cryptomator/mnt/Erasmus_ (see attached pic of Nemo's bookmark tab). 

![Bildschirmfoto vom 2024-11-02 11-51-32](https://github.com/user-attachments/assets/aa216865-4e18-41c8-9b0b-8b60faf5ad0c)

### Steps to reproduce

Unclear what causes this; no activities on my part spotted that cause the creation of a new bookmark.

### Expected behavior

Bookmarks in Nemo are not generated automatically; they are only added manually by user.

**Comment by mtwebster** [comment]
Are those entries under the 'Bookmarks' section of the sidebar? Nothing should be automatically added there that isn't in `~/.config/gtk-3.0/bookmarks` (other than a few standard items - home, filesystem, trash...). This is what nemo monitors for bookmarks, and updates if you add one yourself.

Items in the Device and Network sections are generated by system events - mounts/unmounts, remote connections...

Do these disappear if you restart nemo?

**Comment by Grydpype** [comment]
Thanks for the quick response.

Yes, they are in the 'Bookmarks' section; the other sections are not showing this behavior.

I have deleted all unwanted bookmarks and rebooted; after opening the vault 'Erasmus' during login, a new bookmark called Erasmus appeared in the Nemo bookmarks section automatically as well as in the /gtk-3.0/bookmarks file.
And this happens after every login. Rebooted & logged in 3 times & now I have 3 Erasmus bookmarks in the Nemo desktop and in the config file.

Closing Cryptomator led to the removal of one bookmark; reopening it generated the bookmark again. 

Cheers,
kai.
